service/src/test/java/uk/gov/hmcts/reform/fpl/jobs/UpdateSummaryCaseDetailsTest.java

```diff
@@ -193,6 +195,8 @@ void shouldUpdateCaseWhenSummaryTabInformationIsUpdated() {
             .syntheticCaseSummary(SyntheticCaseSummary.builder()
                 .caseSummaryHasNextHearing("No")
                 .build())
+            .uploadDraftOrdersEventData(UploadDraftOrdersData.builder().build())
```

Comment:
this is a bit weird, it's unrelated: probably the test should not use a real CaseConverter(mapper) but rather a mock. maybe not part of this story.